modules/tensor_mechanics/src/materials/cohesive_zone_model/CZMComputeDisplacementJumpTotalLagrangian.C

```diff
@@ -0,0 +1,117 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "CZMComputeDisplacementJumpTotalLagrangian.h"
+
+registerMooseObject("TensorMechanicsApp", CZMComputeDisplacementJumpTotalLagrangian);
+
+InputParameters
+CZMComputeDisplacementJumpTotalLagrangian::validParams()
+{
+  InputParameters params = CZMComputeDisplacementJumpSmallStrain::validParams();
+  params.addClassDescription(
+      "Compute the displacement jump increment accross a czm interface in local "
+      "coordinates for the Total Lagrangian kinematic formulation");
+
+  return params;
+}
+
+CZMComputeDisplacementJumpTotalLagrangian::CZMComputeDisplacementJumpTotalLagrangian(
+    const InputParameters & parameters)
+  : CZMComputeDisplacementJumpSmallStrain(parameters),
```

Comment:
I thought we decided to derive `CZMComputeDisplacementJumpTotalLagrangian` and `CZMComputeDisplacementJumpSmallStrain` from a same base class. Correct?

Reply:
ups, I forgot about this. 
As requested I added a base class